Enable now CDN for static files

mxstbr opened the pull request:
<!-- FILL OUT THE BELOW FORM OR YOUR PR WILL BE AUTOMATICALLY CLOSED -->
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

**Related issues (delete if you don't know of any)**
Closes #3708

mxstbr committed:
Enable now CDN for static files

brianlovin commented:
Looks like a flow issue with `req` not being in scope - once that's fixed I'll ship this! Awesome stuff @mxstbr :)

mxstbr committed:
Cache static files forever

...no matter if the user's signed in or not since that doesn't actually matter for the file contents!

mxstbr commented:
I'm a dummy, that code was entirely unnecessary on top of that. 🤦‍♂️ Fixed!
